learning_journal/routes.py

```diff
@@ -5,7 +5,7 @@
 def includeme(config):
     """includes our views I guess..."""
     config.add_static_view('static', 'static', cache_max_age=3600)
-    config.add_route("home", "/")
-    config.add_route("edit", "/edit")
+    config.add_route("index", "/")
+    config.add_route("edit_existing", "/edit")
```

Comment:
how are you handling passing the ID of the entry you want to your view?  You'll need to get that information to the view somehow, or you'll never be able to tell which entry it is you are trying to edit.